app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -854,11 +854,10 @@ private void handleInstallButtonClick() {
   }
 
   private void showRecommendsDialog(boolean isLoggedIn, String packageName) {
+    view.showRecommendsDialog();
```

Comment:
Why are we changing this here? Won't we always show the dialog as if the user is logged in regardless of the logic?